ipad.css

```diff
@@ -0,0 +1,332 @@
+.body {
+  max-width: 768px;
+  margin: 0;
+  padding: 0;
+}
+
+.section_first {
+  max-width: 768px;
+}
+
+.nav__link {
+  display: none;
+}
+
+.header {
+  padding: 24px 33px 0px 16px;
+  margin-bottom: 80.32px;
+}
+
+.nav {
+  display: none;
+}
+
+.nav__input {
+  display: block;
+  min-width: 35px;
+}
+
+.title {
+  max-width: 370px;
+  margin-bottom: 48px;
+}
+
+.title__h1 {
+  margin: 0;
+}
+
+.background {
+  max-width: 768px;
+}
+
+.image__car {
+  width: 87%;
+}
+
+.box {
+  margin: 0 auto 63px;
+  width: 63%;
+  flex-wrap: wrap;
+  max-width: 487px;
+}
+
+.box__img-1 {
+  width: 65.92px;
+  height: 50.16px;
+}
+
+.box__img-2 {
+  width: 41.56px;
+  height: 64.45px;
+}
+
+.box__item {
+  margin: 0;
+}
+
+.box__item-1 {
+  margin-right: 66px;
+}
+
+.box__text-number {
+  font-size: 50px;
+  line-height: 61px;
+  font-weight: 600;
+}
+
+.mobile-box {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+  margin-bottom: 55px;
+}
+
+.mobile-test {
+  margin: 0 auto;
+}
+
+.test {
+  margin: 0;
+}
+
+.test__img-3 {
+  margin: 0;
+  margin-right: 9.6px;
+}
+
+.test__text {
+  font-size: 16px;
+  display: inline-block;
+  margin: 0;
+  margin-right: 34.5px;
+}
+
+.main {
+  max-width: 768px;
+}
+
+.future {
+  display: block;
+  background: none;
+}
+
+.section__future-left {
+  max-width: 768px;
+  background: url(img/future.png) no-repeat;
+  background-size: cover;
+}
+
+.future__h1 {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 0;
+  margin-right: 46px;
+  height: 500px;
+  width: auto;
+  white-space: nowrap;
+}
+
+.section {
+  padding-top: 55px;
+}
+
+.section__future {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  max-width: 768px;
+}
+
+.mobile__section {
+  max-width: 300px;
+  margin: 0 116px 60px 34px;
+}
+
+.mobile__section_icon {
+  max-width: 270px;
+}
+
+.section__h1 {
+  margin: 0;
+  margin-bottom: 20px;
+}
+
+.section__h1_safe {
+  max-width: 316px;
+}
+
+.section__p {
+  margin: 0;
+}
+
+.section__icon {
+  margin: 0;
+}
+
+.item__icon {
+  display: flex;
+  align-items: center;
+  margin-bottom: 21px;
+}
+
+.link__section {
+  font-weight: 600;
+  font-size: 16px;
+  margin-right: 37px;
+  margin-left: 0;
+}
+
+.section__info {
+  width: auto;
+  margin: 0 auto;
+  margin-bottom: 52px;
+}
+
+.safe {
+  display: block;
+  position: static;
+}
+
+.section__safe {
+  width: 100%;
+  padding-bottom: 52px;
+}
+
+.section__safe_img {
+  display: none;
+}
+
+.safe__img {
+  display: none;
+}
+
+.section__p_safe {
+  max-width: 316px;
+  margin-bottom: 85px;
+}
+
+.five {
+  display: none;
+}
+
+.mobile__section_flex {
+  display: flex;
+  justify-content: space-around;
+  margin-right: 70px;
+  margin-left: 34px;
+}
+
+@media screen and (max-width: 600px) {
+  .mobile__section_flex {
+    flex-wrap: wrap;
+  }
+
+  .box {
+    margin: 0 auto 34px;
+    max-width: 303px;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)